variables.tf

```diff
@@ -53,6 +53,11 @@ variable "pod_cidr_name" {
   type        = string
 }
 
+variable "gcp_service_account" {
```

Comment:
Needs an empty default value so as not to require changes in existing consumers.